charts/ezkonnect/templates/instrumentedapplications.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: instrumentedapplications.logz.io
+spec:
+  group: logz.io
+  names:
+    kind: InstrumentedApplication
+    listKind: InstrumentedApplicationList
+    plural: instrumentedapplications
+    singular: instrumentedapplication
```

Comment:
You can also set short name, like [here](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/crds/aquasecurity.github.io_vulnerabilityreports.yaml#L15)